students/WWhite/lesson04/Mailroom.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+# -------------------------------------#
+# Desc: Mailroom Part 1
+# Dev: Will White
+# Date: 5/8/2018
+# ChangeLog: (When,Who,What)
+# -------------------------------------#
+
+
+list_donations = [["bill gates", 25000.00, 3], ["monet holt", 50000.00, 2], ["jeff bezos", 123500.09, 3],
+                  ["john wick", 120000.00, 2], ["john snow", 10.56, 1]]  # List of donors
+
```

Comment:
This would be a great place to use a dict, as your main data structure, something like this:
```
donors = {
    'bill gates': [25000.00, 3],
}
```
I think you'll find that this simplifies a lot of your code for looking up and adding donors and donations